Interni/NormeProgetto/components/norme-c2.tex

```diff
@@ -0,0 +1,205 @@
+	\item tracciare riferimenti sulla mole di lavoro in modo da avere una stima dei costi.
+\end{itemize}
+\paragraph{Descrizione}
+I requisiti vengono ricavati da diverse fonti,  tra cui:
+\begin{itemize}
+	\item \textbf{Capitolati d'Appalto}: grazie alla lettura approfondita del capitolato scelto vengono individuati dei requisiti;
+	\item \textbf{Casi d'uso}: requisiti estrapolati da uno o più casi d'uso; 
+	\item \textbf{Verbali}: i requisiti possono emergere sia dalle riunioni interne effettuate dagli \anas{}, sia da contatti o incontri con i rappresentanti dell'azienda proponente.
+\end{itemize}
+\paragraph{Aspettative}
+L'obiettivo dell'attività di analisi è la creazione della documentazione formale contenente tutti i requisiti richiesti dal proponente.
+\paragraph{Struttura}
+La struttura che contraddistingue l'\AdR{} è:
+\begin{itemize}
+	\item \textbf{Descrizione}: contiene informazioni riguardanti il prodotto,  le componenti presenti all'interno del sistema,  la piattaforma d'esecuzione e i vincoli di progettazione individuati;
+	item \textbf{Casi d'uso}: vengono identificati gli attori che interagiscono con le componenti del sistema e le interazioni tra sistema,  attori ed elementi esterni.  Approfondimento nel paragrafo §2.2.4.6;
+	\item \textbf{Requisiti}: rappresentati in una tabella che contiene:
+	\begin{itemize}
+ 		\item tracciamento dei requisiti,  suddivisi in obbligatori,  desiderabili e opzionali,  individuati nel sistema durante l'analisi; approfondimento nel paragrafo §2.2.4.5;
+		\item tracciamento fonte-requisiti e requisiti-fonte. 
+	\end{itemize}
+\end{itemize}
+\paragraph{Classificazione dei requisiti}
+La classificazione dei requisiti verrà effettuata mediante la seguente codifica:\newline
+\centerline{\textbf{R[Tipo][Priorità][Numero]}}\newline
+dove:
+\begin{itemize}
+	\item \textbf{R:} abbrevia "Requisito";
+	\item Tipo: può assumere i seguenti valori:
+	\begin{itemize}
+		\item \textbf{F:} requisito funzionale, ossia la definizione di una particolare caratteristica che deve essere inclusa nel software;
+		\item \textbf{Q:} requisito di qualità, relativi alle regole di qualità del software (efficienza ed efficacia);
+		\item \textbf{V:} requisito di vincolo che rappresenta dei vincoli avanzati dal proponente.
+	\end{itemize}
+	\item \textbf{Priorità:}
+	\begin{itemize}
+		\item \textbf{O:} requisito obbligatorio;
+		\item \textbf{D:} requisito desiderabile;
+		\item \textbf{F:} requisito facoltativo.
+	\end{itemize}
+	\item Numero: codice identificativo dei requisiti.
+\end{itemize}
+\paragraph{Classificazione dei casi d'uso}
+//// DA FARE
+\paragraph{Metriche}
+//// DA FARE
+\subsubsection{Progettazione}
+\paragraph{Scopo}
+L'attività di progettazione definisce, in funzione dei requisiti specificati nel documento \AdR{}, le caratteristiche che il prodotto richiesto deve avere in modo di fornire la soluzione migliore che soddisfi i requisiti degli \glo{stakeholders}.  Questa fase provvede a:
```

Comment:
Cambiare
"... in modo di fornire la soluzione ..."
in
"... in modo da fornire la soluzione ..."

Reply:
Fixed